analysis/tests/parameter_tuning_test.py

```diff
@@ -263,6 +319,42 @@ def test_tune_count(self):
         self.assertEqual(utility_reports[0].metric_errors[0].metric,
                          pipeline_dp.Metrics.COUNT)
 
+    def test_select_partitions(self):
+        # Arrange.
+        input = [(i % 10, f"pk{i/10}") for i in range(10)]
```

Comment:
maybe describe in comment how it looks like to not ask people reading this use their cognitive forces to understand partitions and pids dist?

Reply:
or just hardcode these 10 values